tests/ga/test_monitor.py

```diff
@@ -245,41 +260,9 @@ def test_send_extension_metrics_telemetry_handling_cpu_cgroup_exceptions_errno2(
         ioerror.errno = 2
         patch_cpu_usage.side_effect = ioerror
 
-        CGroupsTelemetry._tracked["/test/path"]= CpuCgroup("cgroup_name", "/test/path")
+        CGroupsTelemetry._tracked["/test/path"] = CpuCgroup("cgroup_name", "/test/path")
 
         PollResourceUsage().run()
         self.assertEqual(0, patch_periodic_warn.call_count)
         self.assertEqual(0, patch_add_metric.call_count)  # No metrics should be sent.
 
-    def test_generate_extension_metrics_telemetry_dictionary(self, *args):  # pylint: disable=unused-argument
```

Comment:
duplicate test which is already tested in cgrouptelemtry class and it's appropriate to test there.